src/components/input/input.js

```diff
@@ -325,7 +328,9 @@ function placeholderDirective() {
     var placeholderText = attr.placeholder;
     element.removeAttr('placeholder');
 
-    inputContainer.element.append('<div class="md-placeholder">' + placeholderText + '</div>');
+    var placeholder = '<div class="md-placeholder" ng-click="delegateClick()">' +
```

Comment:
This is only a fallback for when the `placeholder` attribute is present, so it won't even be there otherwise. All it's doing is delegating a click to the input, and it worked everywhere I checked (multiple versions of IE, Firefox, Chrome, Safari, Mobile Safari). I need to check Android, but given how minor this thing is, I don't see how it would be an issue.